.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  ELM_VERSION: 0.19.1
+  NODE_VERSION: 10.8.0
+  NPM_VERSION: 6.4.0
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set version variables
+      id: versions
+      run: |
+        echo "node_version=$NODE_VERSION" >> $GITHUB_OUTPUT
+        echo "npm_version=$NPM_VERSION" >> $GITHUB_OUTPUT
+
+    - name: Install elm
+      run: |
+        curl -L -o elm.gz https://github.com/elm/compiler/releases/download/$ELM_VERSION/binary-for-linux-64-bit.gz
+        gunzip elm.gz
+        chmod +x elm
+        sudo mv elm /usr/local/bin/
+    
+    - name: Install volta (node version manager)
```

Comment:
I'm not opposed, but why not use `nvm`, since it's the standard tool?

I actually added `nvm` in my other MR, https://github.com/kfrn/reel-time/pull/24 - that was before I read through this MR, sorry!

Would you be willing to update this MR to use `nvm`? If not, I'm happy to make the change!

Reply:
Thanks for the comprehensive reply, @ingk!

I'm undoubtedly a bit blinded by using Linux at work and at home, and now being so used to using tools like `nvm`. But I well remember the frustrations I encountered when I was new to all this. And I'm still sometimes caught out by forgetting to run `nvm install`/`nvm use` when switching projects.

So yes, I'm happy to go with Volta! Maybe someone else will come along and want to make another translation. :grinning: If you want to update this MR, then I'll approve.